pkg/graveler/ref/manager.go

```diff
@@ -140,10 +174,7 @@ func (m *KVManager) ListRepositories(ctx context.Context) (graveler.RepositoryIt
 func (m *KVManager) DeleteRepository(ctx context.Context, repositoryID graveler.RepositoryID) error {
 	// TODO: delete me
 	// temp code to align with DB manager. Delete once https://github.com/treeverse/lakeFS/issues/3640 is done
-	_, err := m.GetRepository(ctx, repositoryID)
-	if errors.Is(err, kv.ErrNotFound) {
-		return graveler.ErrRepositoryNotFound
-	}
+	_, err := m.getRepository(ctx, repositoryID)
 	if err != nil {
 		return err
 	}
```

Comment:
what is the below TODO?

Reply:
Delete flow will be addressed in https://github.com/treeverse/lakeFS/issues/3713